Update exercises.md

aballet opened the pull request:
Adrien BALLET and Sébastien MOTTET added to the matrix multiplication exercise

Commit:
Update exercises.md

Adrien BALLET and Sébastien MOTTET added to the matrix multiplication exercise

Commit:
Update exercices.md

Henri DEVILLEZ added to the "Tri par insertion" exercise.

Zashas commented:
Pas de PR groupé s'il vous plait, chacun doit être capable de faire un PR individuellement. Vous devrez de toute façon par la suite en faire un chacun pour soumettre votre code.

aballet commented:
Désolé. J'en refais un individuel alors?

Commit:
Demande pour la multiplication de matrices

Cette demande est individuelle. J'ai enlevé les noms des personnes avec qui j'avais fait la précédente demande groupée.
